### ISSUE TITLE
Commas in arrays of string literals are stripped

### ISSUE THREAD

**Issue by gabegorelick** [spicyj/es3ify#10]
```js
it('should not strip commas from arrays of string literals', function() {
    expect(transform('["a, b, c"]'))
        .toEqual('["a, b, c"]'); // equals '["a b c"]' instead
});
```

**Comment by spicyj** [comment]
Well that was clowny. Fix published as 0.2.1.

**Comment by spicyj** [comment]
(Thanks!)
